# HG changeset patch
# User Rik <rik@octave.org>
# Date 1416682194 28800
#      Sat Nov 22 10:49:54 2014 -0800
# Node ID 718aaf1e84f799150b9ebf440f17839d9c9e8755
# Parent  fcd9e1198847a1cc00bf98a44ae534b5f5ded402
kron.cc: Use column vector in undeprecated Array constructor (cset 1f4455ff2329).

* kron.cc: Use column vector in undeprecated Array constructor
(cset 1f4455ff2329).

diff --git a/libinterp/corefcn/kron.cc b/libinterp/corefcn/kron.cc
--- a/libinterp/corefcn/kron.cc
+++ b/libinterp/corefcn/kron.cc
@@ -133,17 +133,17 @@ kron (const MSparse<T>& A, const MSparse
 
 static PermMatrix
 kron (const PermMatrix& a, const PermMatrix& b)
 {
   octave_idx_type na = a.rows ();
   octave_idx_type nb = b.rows ();
   const Array<octave_idx_type>& pa = a.col_perm_vec ();
   const Array<octave_idx_type>& pb = b.col_perm_vec ();
-  Array<octave_idx_type> res_perm (dim_vector (1, na * nb));
+  Array<octave_idx_type> res_perm (dim_vector (na * nb, 1));
   octave_idx_type rescol = 0;
   for (octave_idx_type i = 0; i < na; i++)
     {
       octave_idx_type a_add = pa(i) * nb;
       for (octave_idx_type j = 0; j < nb; j++)
         res_perm.xelem (rescol++) = a_add + pb(j);
     }
 
